project/AppDependencies.scala

```diff
@@ -30,13 +30,13 @@ object AppDependencies {
   )
 
   val test: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-test-play-28"   % bootstrapPlay28Version  % Test,
+    "uk.gov.hmrc"             %% "bootstrap-test-play-28"   % bootstrapPlay28Version                % Test,
```

Comment:
Very minor one: We don't notice any relevant changes here apart from the spacing?

Reply:
The spaces have been removed

Reply:
Has it? I still see it? I don't want to drag you too much because of this, the rest looks fine

<img width="852" alt="image" src="https://user-images.githubusercontent.com/108010791/182158031-074711ab-8770-46cb-bc99-b69963897333.png">

Reply:
I did change it not sure how that happened.